Sources/Model/TransferMethod/HyperwalletTransferMethodQueryParam.swift

```diff
@@ -20,11 +20,14 @@ import Foundation
 
 /// Representation of the common transfer method's query parameters.
 public class HyperwalletTransferMethodQueryParam: QueryParam {
-    private enum QueryParam: String {
+   public enum QueryParam: String {
         case status
+        case type
     }
+
     /// Returns transfer method with this status.
     public var status: QueryStatus?
+    public var type: HyperwalletTransferMethod.TransferMethodType?
```

Comment:
You have to add swift comments if you make anything public

Reply:
Okay.done